### PR TITLE
tf redis move to module and enable ingress

### PR DESCRIPTION
**Pull request by suryaoruganti**

### CODE REVIEW

File: modules/aws/elasticacheRedis/outputs.tf [hunk 1]
```diff
@@ -1,3 +1,45 @@
...
+  value       = module.redis.endpoint
+  description = "Redis primary endpoint"
+}
+
+output "cluster_host" {
+  value       = module.redis.host
+  description = "Redis hostname"
+}
+
+// output "cluster_security_group_id" {
```
Comment (admiralnico) [inline modules/aws/elasticacheRedis/outputs.tf:32]: do you want to delete the commented out parts?

File: modules/aws/elasticacheRedis/main.tf [hunk 2]
```diff
@@ -1,9 +1,57 @@
...
-  cluster_id           = var.name
-  engine               = "redis"
-  node_type            = var.node_type
-  num_cache_nodes      = var.num_cache_nodes
-  parameter_group_name = var.parameter_group_name
-  engine_version       = var.engine_version
-  port                 = 6379
-}
+module "redis" {
+  source = "github.com/cloudposse/terraform-aws-elasticache-redis.git"
```
Comment (admiralnico) [inline modules/aws/elasticacheRedis/main.tf:2]: is cloudposse a better source for this module?

Comment (suryaoruganti) [reply modules/aws/elasticacheRedis/main.tf:2]: that is the only source for the module. But one catch - need to add a specific version tag to this.

Comment (admiralnico) [reply modules/aws/elasticacheRedis/main.tf:2]: Can you add a tag or commit id? that would be better

File: modules/aws/elasticacheRedis/outputs.tf (hunk 1 above)
Comment (suryaoruganti) [reply modules/aws/elasticacheRedis/outputs.tf:16]: removed most of them that aren't relevant

File: modules/aws/elasticacheRedis/main.tf (hunk 2 above)
Comment (suryaoruganti) [reply modules/aws/elasticacheRedis/main.tf]: done
can you approve so I can merge

**Pull request merged**